chapter_linear-networks/linear-regression.md

```diff
@@ -391,7 +391,7 @@ rather than writing costly for-loops in Python.**)
 %matplotlib inline
 from d2l import mxnet as d2l
 import math
-import numpy as np
+from mxnet import np
```

Comment:
Can we keep using `import numpy as np` like what we do for pytorch and tf? This will simplify the implementation (tab all) and we don't need any change in this section

Reply:
I changed this before to fix the problem [here](https://github.com/d2l-ai/d2l-en/pull/2047/commits/0c092b9dff63f2c090b4bc761847da9e987d3ddf). But I found this as a better implementation in terms of consistency. Either we should use `from mxnet import np` across the book (which is the case right now) or change everything to `import numpy as np`.

Reply:
Sure. Let's use mxnet.np here for consistency throughout the book.